src/main/xar-resources/error-page.html

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<div xmlns="http://www.w3.org/1999/xhtml" class="templates:surround?with=templates/page.html&amp;at=content">
+<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
     <h1>An error has occurred</h1>
     <p>An error has been generated by the application.</p>
-    <pre class="error templates:error-description"/>
+    <pre class="error" data-template="templates:error-description" />
     <div class="source-links">
-        <p>View source: <a href="login.html" class="templates:load-source">this page</a>.</p>
+        <p>View source: <a href="login.html">this page</a>.</p>
```

Comment:
This entire div should be removed, since without the `templates:load-source` the link won't be transformed into the link to eXide. See the corresponding change in [https://github.com/eXist-db/function-documentation/pull/41/commits/65f05d6f964c66c7ae8357a562a61900fa3de4f9](function-documentation) and [documentation](https://github.com/eXist-db/documentation/pull/626/commits/387db1971174a6a1037922ae4dd84ae264db1469).